src/components/switchComponent/index.tsx

```diff
@@ -1,17 +1,49 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
+import { FlexboxProps } from '@chakra-ui/styled-system';
 import {
   Switch as ChakraSwitch,
   SwitchProps as ChakraSwitchProps,
+  FormControl,
+  FormLabel,
 } from '@chakra-ui/react';
 
 export type SwitchProps = Pick<
   ChakraSwitchProps,
-  'onChange' | 'isChecked' | 'variant'
->;
+  'onChange' | 'isChecked' | 'variant' | 'isDisabled'
```

Comment:
can we issue a BREAKING CHANGE and remove variant? I think we should also create a tech task to remove variant completely and move to radio button in the dev overlay. Switch represents a boolean while themeSwitcher is more a radio button.